service/src/test/java/bio/terra/workspace/common/utils/MockMvcUtils.java

```diff
@@ -0,0 +1,88 @@
+package bio.terra.workspace.common.utils;
+
+import bio.terra.workspace.service.iam.AuthenticatedUserRequest;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.test.web.servlet.request.MockHttpServletRequestBuilder;
+
+/**
+ * A collection of utilities and constants useful for MockMVC-based tests. This style of tests lets
+ * us test controller-layer code (request/response parsing, authz, and validation) without actually
+ * spinning up a local server.
+ *
+ * <p>TODO: it's probably worth looking into whether we can automatically pull routes from the
+ * generated swagger, instead of manually wrapping them here.
+ */
+@Component
+public class MockMvcUtils {
```

Comment:
I think some endpoints will end up needing connected tests instead of just unit tests so I'm hesitant to include that in the name, and I'd lean towards `MockMvcUtils` over something like `ControllerTestUtils` since these are really testing endpoints, not just the controller code